categories.md

```diff
@@ -0,0 +1,27 @@
+---   
+layout: post
+title: "Categories"
+permalink: /categories/
+---
+
+<div id="archives">
+ <ul>
+{% for category in site.categories %}
+  <div class="archive-group">
+    {% capture category_name %}{{ category | first }}{% endcapture %}
```

Comment:
Also just wondering why you decided to create a new variable `category_name` instead of using `category` (created in line 9)?